internal/runner/service.go

```diff
@@ -803,8 +816,8 @@ func convertToMonitorEnvStoreResponse(msg *runnerv1.MonitorEnvStoreResponse, sna
 			OriginalValue: item.Value.Original,
 			ResolvedValue: item.Value.Resolved,
 			Status:        status,
-			CreateTime:    item.Var.Created.String(),
-			UpdateTime:    item.Var.Updated.String(),
+			CreateTime:    item.Var.Created.Format(time.RFC3339),
```

Comment:
@sourishkrout let's use the RFC3339 which is more standard for sharing dates through systems and compatible with ISO8601, It was tested with the extension and is working fine.